### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by Disane87** [benmarch/angular-ui-tour#79]
**I am using the exact versions of the following libraries:**

AngularJS: 1.5.0
Angular Bootstrap: 1.1.2
Angular UI Tour: 0.6.1

**I have installed this library via:**
manually via download

**I have observed the following behavior:**
I have a multi-lingual page with german and english translations. So I need to translate the tour step texts as well, but that doesn't work for me.

I declared them like this:

``` html
  <div class="col-sm-3"
    tour-step tour-step-belongs-to="myDetachedTour"
    tour-step-title="{{TUTORIAL_TOUR_STEP_WEATHER_TITLE | translate }}"
    tour-step-content="{{TUTORIAL_TOUR_STEP_WEATHER_TEXT | translate}}"
    tour-step-order="0"
    >
```

There is no text at all:
![bildschirmfoto 2016-09-17 um 22 40 39](https://cloud.githubusercontent.com/assets/956158/18610860/d0e08a14-7d27-11e6-93dc-81ddedfe2bf9.png)

Additionally the buttons are always in english.

**This is how I expected it to behave:**

I need a way to translate all the texts for the tour steps. I use Pascal Prechts translation for Angular.

**Comment by benmarch** [comment]
Hey @Disane87 sorry for the delayed response. One thing I noticed is that you are using Angular Bootstrap 1.x and Angular UI Tour 0.6.x which are not compatible. Try upgrading Angular Bootstrap to 2.x or downgrading Angular UI Tour to 0.5.x. If you still experience this issue, please let me know.

**Comment by VanHumbeeckA** [comment]
Hi there,

We are seeing the same kind of issues, with the 'content'-part in a ui-tour-directive.
(The 'title'-part of the tour directive is translated ok)
We are using angular-bootstrap 2.X and angular-ui-tour 0.6.x

Inside the object returnd by tour.init(), the value is translated ok, but it is not shown in the DOM.
Sometimes it is just the original translation. Or even worse: the keys only (never translated)

**Comment by skalamichal** [comment]
Hi, this may help, I had similar issue with content update. I had to update the tour-step-directive with following code:

`
// watch for content updates
   contentWatch = attrs.$observe(TourHelpers.getAttrName('content'), function(content) {
      if (content) {
         step.trustedContent = $sce.trustAsHtml(step.content);
      }
   });
`

I put it just after the enabledWatch, before TourHelpers part. You may need to add the observer for the title as well.

Michal

**Comment by benmarch** [comment]
Sorry for ignoring this issue for so long, @Disane87. This will be fixed in the next release (0.7.1).

**Comment by benmarch** [comment]
Version 0.8.0 has been released, please verify that this issue has been resolved. Note that there are a few breaking changes that might need to be addressed: https://github.com/benmarch/angular-ui-tour#versions.
